vaadin-details.ts

```diff
@@ -0,0 +1 @@
+import './theme/lumo/vaadin-details';
```

Comment:
Now that the source is in esm, should the root level exports be done from an index.ts entry point file instead?

Reply:
What would it look like in case of repos with multiple components (grid, text-field etc)? 
Do we replace `all-imports.html` with `index.ts` in those cases?

If you think it makes sense, I'm ok to rename the root file.

Reply:
Fair point, we don't necessarily always want to import all the modules from a multi-component repo, or require `import '@vaadin/vaadin-grid/theme/lumo/vaadin-grid-tree-column'`